### PR TITLE
[new release] merge-fmt (0.3)

### PR DESCRIPTION
**Pull request by hhugo**
Git mergetool leveraging code formatters

- Project page: <a href="https://github.com/hhugo/merge-fmt">https://github.com/hhugo/merge-fmt</a>
- Documentation: <a href="https://hhugo.github.io/merge-fmt/">https://hhugo.github.io/merge-fmt/</a>

##### CHANGES:

- Add support of dune formatter. (hhugo/merge-fmt#3)

### CODE REVIEW

File: packages/merge-fmt/merge-fmt.0.3/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer:   "hugo.heuzard@gmail.com"
+authors:      "Hugo Heuzard"
+homepage:     "https://github.com/hhugo/merge-fmt"
+doc:          "https://hhugo.github.io/merge-fmt/"
+bug-reports:  "https://github.com/hhugo/merge-fmt/issues"
+dev-repo:     "git+https://github.com/hhugo/merge-fmt.git"
+license:      "MIT"
+
+build: [["dune" "build" "-p" name "-j" jobs]]
```
Comment (kit-ty-kate) [inline packages/merge-fmt/merge-fmt.0.3/opam:10]: Reading the source code I'm noticing that there are some tests. Do you want to run them?
```suggestion
build: [["dune" "build" "-p" name "-j" jobs]]
run-test: [["dune" "runtest" "-p" name "-j" jobs]]
```

Comment (hhugo) [reply packages/merge-fmt/merge-fmt.0.3/opam:10]: I don't think so, they are too fragile, they depend on `git` output.

Comment (kit-ty-kate) [reply packages/merge-fmt/merge-fmt.0.3/opam:10]: ok fair enough

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**